style.css

```diff
@@ -538,3 +531,251 @@ body {
   border-radius: 8px;
   border: 1px solid #cfd8dc;
 }
+
+#submit:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+#submit:active {
+  background: #2230d2;
+  color: #fff;
+}
+
+#submit:disabled {
+  background: #c1c7d0;
+  color: #fff;
+}
+
+.second {
+  display: none;
+}
+
+.two {
+  display: none;
+}
+
+@media screen  and (min-width: 768px) {
+  body {
+    background: #e5e5e5;
+  }
+
+  .home-page-section {
+    background: #fff;
+    height: 916px;
+    width: 1440px;
+    display: flex;
+    background-image: url(/assets/desktop-version/Header\ bg.svg);
```

Comment:
- [ ] Kindly fix the typos here, background image display. Right now, it doesn't show up on the page.